.travis.yml

```diff
@@ -18,7 +18,7 @@ env:
     - PLONE_VERSION=4.3
     - PLONE_VERSION=4.3 DEXTERITY_ONLY=true
     - PLONE_VERSION=4.3 JQUERY=1.9.1.3
-    - PLONE_VERSION=4.3 NEW_WIDGETS=true
```

Comment:
@rodfersou nice catch! I want a separated commit for this one and an update of the documentation.